### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by bblaszkow06**
The workflow testing the orb's jobs can be seen here:
https://app.circleci.com/pipelines/github/membraneframework/membrane_file_plugin?branch=test-orb&filter=all
https://github.com/membraneframework/membrane_file_plugin/compare/test-orb

### CODE REVIEW

File: src/commands/run_dialyzer.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+description: >
+  Runs dialyzer using cache for PLT files.
+  Assumes Elixir & Erlang are installed via asdf-vm
+parameters:
+  version:
```
Comment (varsill) [inline src/commands/run_dialyzer.yml:5]: how about renaming it into `cache-version`, to keep consistency?

File: src/commands/use_build_cache.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+description: >
+  Allows to run compilation using cache
+parameters:
+  version:
```
Comment (varsill) [inline src/commands/use_build_cache.yml:4]: The same goes here - perhaps `cache-version` would fit better here?

File: src/scripts/greet.sh [hunk 3]
```diff
@@ -1,2 +0,0 @@
-#!/bin/bash
-echo Hello "${PARAM_TO}"
```
Comment (varsill) [inline src/scripts/greet.sh:2]: Perhaps we shouldn't leave an empty `scripts/` directory (as far as I can see it is not the standard directory name for the scripts directory, as you need to call a script with `include(<path_to_script>)`)

Comment (bblaszkow06) [reply src/scripts/greet.sh:2]: IIUC, you want to remove `scripts` dir completely, right? Then it does exactly that, git has no concept of catalogues/folders, it only stores files. The only way to store an 'empty' directory is to put an empty file inside it - by convention named `.gitkeep`

**Review by varsill — APPROVED** [review]
lgtm 🥇

**Pull request merged**